src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -157,6 +162,11 @@ - (instancetype)initWithHost:(NSString *)host
     _requestWriter = requestWriter;
 
     _requestHeaders = [[GRPCRequestHeaders alloc] initWithCall:self];
+
+    if ([requestWriter isKindOfClass:[GRXImmediateSingleWriter class]]) {
+      _unaryCall = YES;
+      _unaryOpBatch = [NSMutableArray arrayWithCapacity:6];
```

Comment:
Please replace magic number "6" with a readable constant, and add comment explaining why 6.